src/devices/Ft4222/FtCommon.cs

```diff
@@ -24,23 +25,33 @@ public static List<DeviceInformation> GetDevices()
             FtStatus ftStatus = 0;
 
             // Check device
-            uint numOfDevices = 0;
-            ftStatus = FtFunction.FT_CreateDeviceInfoList(ref numOfDevices);
+            uint numOfDevices;
+            ftStatus = FtFunction.FT_CreateDeviceInfoList(out numOfDevices);
 
             Debug.WriteLine($"Number of devices: {numOfDevices}");
             if (numOfDevices == 0)
                 throw new IOException($"No device found");
 
-            byte[] sernum = new byte[16];
-            byte[] desc = new byte[64];
-            uint flags = 0;
-            for (int i = 0; i < numOfDevices; i++)
+            for (uint i = 0; i < numOfDevices; i++)
```

Comment:
unless there is any reason for this being `uint` I'd move this back to `int`

Reply:
The function I'm passing i is taking unit, that does avoid a cast.